Automattic/pocket-casts-android
# Add new widgets

## Description

This PR integrates widgets branch without our main branch. There are no functional changes. Almost everything was developed on separate branches and targeted `task/widgets`.

There is one change. Update for the "What's New" version code so it is displayed.

Closes #1992 

> [!warning]  
> Do not merge this until #2038, #2040, and #2041 are merged into `task/widgets`.

## Testing Instructions

1. Install the app from d1c9edbaab0ddb4aa98e9786d19e2e0bea359321.
2. Update the app with the one from this `task/widgets` branch.
3. Open the app.
4. You should be greeted with the "What's New" popup.
5. Close the app.
6. Open the app again.
7. You should not see the popup again.

---

1. Do a clean install from `task/widgets` branch.
2. Open the app.
3. You should not see the "What's New" popup.

## Screenshots or Screencast 

![321209536-41ec6bc8-332b-4aff-a6f4-c60fb2e994e3](https://github.com/Automattic/pocket-casts-android/assets/30936061/7c18930f-c006-410b-a5c7-6696a19f35e7)

## Checklist
 
#### I have tested any UI changes...
